python/hail/methods/impex.py

```diff
@@ -1076,8 +1076,8 @@ def import_matrix_table(paths, row_fields={}, key=[], entry_type=tint32, missing
     If row field names are missing (or the no_header flag is ``True``), row fields
     are imported in column-order as `f0`, `f1`, ... (0-indexed).
 
-    If the no_header flag is ``True``, column IDs will be imported as `col0`,
-    `col1`, ... (also 0-indexed).
+    If the no_header flag is ``True``, column IDs will be imported as integers
```

Comment:
For consistency with parameters below:
```
If ``no_header=True``, column IDs are set to integers 0, 1, ... (also 0-indexed) in column order.
```

Reply:
What is the name of the column ID (key) field?

More generally, the docs don't make it easy to figure out the full schema of the resulting matrix table.  I think it would help to document the resulting schema more directly like we do in the other import methods that create matrix tables. You could also show the resulting row, col, entry schema in the first example.